lms/djangoapps/teams/tests/test_views.py

```diff
@@ -3017,3 +3017,29 @@ def test_upload_non_ascii(self):
             [user.username for user in team.users.all()],
             [user_name]
         )
+
+    def test_upload_assign_masters_learner_to_non_protected_team(self):
+        """
+        Scenario: Attempt to add a learner enrolled in masters track to an existing, non-org protected team.
```

Comment:
okay, so this makes sense. out of curiosity, what would happen / do we have a case where we try to add them to a team that exists that's not protected and also has users in it already? will that fall into the ferpa bubble error message?

Reply:
i imagine the ferpa bubble case is for when we are trying to add mixed users to a nonexistant team, right?